src/app/shared/components/dropdown/dropdown.component.ts

```diff
@@ -23,12 +23,12 @@ export class DropdownComponent {
   /**
    * Current option selected.
    */
-  @Input() selection: string;
+  @Input() selection: string[];
```

Comment:
Why are you changing this to a multi-valued input? Dropdown will only ever have one selected thing.